### ISSUE TITLE
HTML

### ISSUE THREAD

**Issue by pnrobinson** [P2GX/maxodiff#80]
Let's not use a text entry field to show the HPOs, we do not want to allow users to change them here (they need to change the phenopacket!)
Therefore, in the template, we need to replace this:
```java
<td><input type="hidden" th:field="*{presentHpoTermIds}" readonly="readonly"/></td>
``` 
with something like this
```java
<td>
      <ul>
          <li th:each="entry : ${observedHpoTerms}">
              <!-- entry.key = id, entry.value = label -->
              <a th:href="@{https://hpo.jax.org/term/{id}(id=${entry.key})}"
                 th:text="${entry.value} + ' (' + ${entry.key} + ')' "
                 target="_blank"
                 rel="noopener noreferrer">
              </a>
          </li>
      </ul>
  </td>
```
This will create a list of observed (present) terms with links out to the HPO webpage.
I think that we need to create a map in the java and add the map to the Thymeleaf template, but I am not sure how to do that.
```java
Map<String,String> observedHpoTerms ...
```
the key is the HPO term label, the value is the TermId - both need to be represented as Strings.

**Comment by bhujbal-aarya** [comment]
@pnrobinson  pleaseassign me this issue
